src/AttributesAndMethods.jl

```diff
@@ -71,19 +103,35 @@ struct map_from_weil_divisors_to_class_group
 end
 export map_from_weil_divisors_to_class_group
 
+
+"""
+    map_from_weil_divisors_to_class_group( v::NormalToricVariety )
+
+Computes the map from the Weil divisors to the Class group of the normal toric variety `v`.
+"""
 function map_from_weil_divisors_to_class_group( v::NormalToricVariety )
     gap_MapFromWeilDivisorsToClassGroup = GAP.Globals.MapFromWeilDivisorsToClassGroup( v.GapNTV )
     return map_from_weil_divisors_to_class_group( gap_MapFromWeilDivisorsToClassGroup )
 end
 export map_from_weil_divisors_to_class_group
 
 
+"""
+    dimension( v::NormalToricVariety )
```

Comment:
everywhere else in Oscar, we are using `dim` right now.

Reply:
Ok. I will change this to `dim`.